lib/v3/parser.js

```diff
@@ -385,8 +386,7 @@ class Parser extends EventEmitter {
                     }
 
                     if (declaration.type === 'FunctionDeclaration') {
-                        const func = this.parseFunctionDeclaration(declaration);
-
+                        const func = this.parseFunctionDeclaration(declaration, exportNodeComment.keywords);
```

Comment:
`parseFunctionDeclaration` method used there only for parse JS AST nodes and provide low-level information that can be extracted only from JS AST. And then I use `emitMethodItem` method that use contextual information and fill additional data to item. So, I prefer to move `return` parsing logic to `emitMethodItem` method, that should cover all usages of the code and you is not need to extend method, becase second method is already has information about comment with all parsed keywords.

Reply:
Yes of course, it is a better separation of concerns.